style.css

```diff
@@ -0,0 +1,6 @@
+
+h1  {
```

Comment:
Even though your implementation works as expected and does change the h1 tag color attribute 🔥, the task points out a specific requirement. 
_A .css file that includes **one class** with styles for your header (make it in your favorite color)._
Kindly make change your CSS selector into a class and don't forget to use it on your `header` or `h1` elements 🧐